From: Tormod Volden <debian.tormod@gmail.com>
Date: Sat Jan  8 23:22:19 AM CET 2022
Subject: [PATCH] man pages: Fix misspelled BR macro
Forwarded: 2022-01-09

Index: xscreensaver-git/hacks/xscreensaver-getimage-video.man
===================================================================
--- xscreensaver-git.orig/hacks/xscreensaver-getimage-video.man
+++ xscreensaver-git/hacks/xscreensaver-getimage-video.man
@@ -37,7 +37,7 @@ Print diagnostics.
 .BR vidcat (1),
 .BR vidtomem (1),
 .BR vrlctl (1),
-.bR motioneye (1).
+.BR motioneye (1).
 .SH COPYRIGHT
 Copyright \(co 2001-2021 by Jamie Zawinski.  Permission to use, copy,
 modify, distribute, and sell this software and its documentation for
